### ISSUE TITLE
I need help 

### ISSUE THREAD

**Issue by Glitchin0** [ultrasecurity/Storm-Breaker#204]
okay so i downloaded everything probably but whenever i try to use geolocation "nearyou" or "weather" to get the location of the person who open the link it doesn't give option to allow or deny when i open localhost/2525 and log in and try to use the link that it give "http://localhost:2525/templates/nearyou/index.html" it work FINE but of course this is just for me to use local host... so when i use ngrok and do "ngrok http 2525" and then i use one of the 2 links it give me and log in then i try to use any of the links they don't give info and the geolocation site doesn't even give request to allow or deny permission... and i tried to use my own domain and and it does the same.. the geolocation page load "nearyou" and then i click on continue but then it say "Please Refresh This Page and Allow Location Permission..." and failed instead of continue.. and it doesn't show request to gain permission and when i go to the site setting to try and enable the location setting it doesn't let me it just says "Blocked" and i cant edit it.. please help... idk if this confusing to you sir you can just add me on discord and i can show you what i mean in a call i can share screen or send you a video.. my discord is "𝙂𝙡𝙞𝙩𝙘𝙝#6763"

Summary: there is a bug in the program .. on localhost it work fine but in ngrok or my personal domain it doesnt work..  Please help.. thanks <3 my discord "𝙂𝙡𝙞𝙩𝙘𝙝#6763"

![Screenshot_5](https://user-images.githubusercontent.com/115073981/202796330-6261849e-5356-4cd5-8042-0ab912677ff8.png)
![Screenshot_6](https://user-images.githubusercontent.com/115073981/202796335-b135a01e-6f1d-454d-a723-cca48fa685fd.png)
![Screenshot_7](https://user-images.githubusercontent.com/115073981/202796336-de5f6196-bfdd-4f29-911c-d3448acf64bf.png)

![Screenshot_4](https://user-images.githubusercontent.com/115073981/202796323-1ffbbef9-87dd-49ec-845f-f0f78e928f3f.png)

![Screenshot_1](https://user-images.githubusercontent.com/115073981/202796309-d2e3e7d7-e57c-40b9-8eeb-9460e2595806.png)
![Screenshot_2](https://user-images.githubusercontent.com/115073981/202796313-92bb0cae-95a4-41fe-96cb-efc4def57257.png)
![Screenshot_3](https://user-images.githubusercontent.com/115073981/202796320-8f9763ef-b171-4322-b759-725108280b38.png)

**Comment by joshkar** [comment]
Hello. We are looking into this matter today... We apologize for the delay in responding

**Comment by joshkar** [comment]
Ability. geo location api based on javascript. Requires the use of https. Does the link you open have such an item?

**Comment by Glitchin0** [comment]
yeah it does i think, i mean the ngrok one is the one you recommend and its not working also "ssh -R localhost.run" method not working  so is there a way to make it over wan or public i mean?? i have a website with a domain can i inject the script in some how maybe? 

also sorry for the delay i kinda got lose on track

**Comment by Glitchin0** [comment]
okay i think that solved it, because when i log in the web panel the link the get provided from the panel is http so i had to manually change it to https thats why it wasn't working, maybe in future you can change the script to make a https instead of http by itself just so no one else fall in same issue but thanks anyways for the help and sorry for the late reply <3

**Comment by Glitchin0** [comment]
but there is new problem now, the method doesn't work on IOS like it give info but when they click allow location they get failed and i don't get the location at all
